src/index.html

```diff
@@ -5,8 +5,292 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="#">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,500;1,500&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+      href="#"
+      id="hederlink"
+      class="logo">
+        <img
+        src="./images/logo.png"
+        alt="imeg"
+        class="logo_img"
+        >
+      </a>
+      <nav class="nav">
+        <a
+        href="#"
+        class="is-active nav__link"
+        >
+          apple
+        </a>
+        <a
+        href="#"
+        class="nav__link"
+        >
+          samsung
+        </a>
+        <a
+        href="#"
+        class="nav__link"
+        >
```

Comment:
```suggestion
        <a
          href="#"
          class="nav__link"
        >
```
Форматування важливе. Його потрібно дотримуватись